### PR TITLE
Add episode 55

### PR DESCRIPTION
**Pull request by colebow**

### CODE REVIEW

File: _episodes/55.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+* Improved performance for Delta Lake queries without table statistics
+* Improved performance for Iceberg queries with filters on `ROW` columns
+
+[Trino 438](https://trino.io/docs/current/release/release-438.html)
+
+* Support for access control with Open Policy Agent
+* Support for `ALTER COLUMN ... DROP NOT NULL` in Iceberg and PostgreSQL
+* Support for configuring page sizes in Delta Lake, Hive, and Iceberg
+* Better type support for the `reduce_agg()` function
+
```
Comment (mosabua) [inline _episodes/55.md:47]: Additional note here to https://trino.io/blog/2024/02/06/opa-arrived

File: _episodes/55.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+## Concept of the episode: Peaka
+
+Another Trino Community Broadcast episode means another cool piece of technology
+that works with Trino for us to show off to the community. This time it's Peaka,
```
Comment (mosabua) [inline _episodes/55.md:51]: that uses Trino ?

File: _episodes/55.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+a no-code approach to date warehousing that makes it easier than ever to set up
+your data stack without needing a ton of complex engineering.
```
Comment (mosabua) [inline _episodes/55.md:53]: you are not really setting up a stack .. Peaka kinda is the stack .. right? Apart from data sources I guess

File: _episodes/55.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+In [their own words](https://www.peaka.com/docs/getting-started/what-is-peaka/),
+Peaka is a platform that merges disparate data sources into a single data layer,
+letting you join and blend them, query them using SQL or natural language, and 
+expose your data to outside users through APIs. Sounds a bit like Trino, right?
+That's because underneath the hood, Trino is a key part of how they're making it
+happen. In this episode, we talk to the team at Peaka about where they got
+started, how they're making it easier than ever to leverage the federation that
+Trino is capable of, and the work they've done on top to integrate their
+platform with ever SaaS data source under the sun.
```
Comment (mosabua) [inline _episodes/55.md:63]:
```suggestion
platform with every SaaS data source under the sun.
```

Comment (mosabua) [reply _episodes/55.md:63]: also .. many other data sources

File: _episodes/55.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+If you want to see what the platform is like, then look no further. We'll be
+exploring:
+
+* Connecting to data sources
+* Filtering and combining data
+* Editing and running queries, including their visual query editor
+* ...and more! (TBD once we have actually streamed the episode)
+
+## PR of the episode: #18719: Alluxio cache
```
Comment (mosabua) [inline _episodes/55.md:75]: Filesystem caching with Alluxio

File: _episodes/55.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+
+Perhaps it's a little easier to link to the issue for tracking
+[the rollout](https://github.com/trinodb/trino/issues/20550), but however you
+want to present it, caching in Trino is here! Caching is a huge performance win
```
Comment (mosabua) [inline _episodes/55.md:79]:
```suggestion
want to present it, caching in Trino is renewed! Caching is a huge performance win
```

File: _episodes/55.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+for a wide variety of use cases, allowing the engine to run faster, better, and
+pump out query results at an unparalleled pace. This is going to lead to huge
```
Comment (mosabua) [inline _episodes/55.md:81]:
```suggestion
pump out query results at an unparalleled pace. This is going to lead to 
```

File: _episodes/55.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+* ...and more! (TBD once we have actually streamed the episode)
+
+## PR of the episode: #18719: Alluxio cache
+
+Perhaps it's a little easier to link to the issue for tracking
+[the rollout](https://github.com/trinodb/trino/issues/20550), but however you
+want to present it, caching in Trino is here! Caching is a huge performance win
+for a wide variety of use cases, allowing the engine to run faster, better, and
+pump out query results at an unparalleled pace. This is going to lead to huge 
+performance improvements for Trino queries that can leverage it, and you'll hear
```
Comment (mosabua) [inline _episodes/55.md:82]:
```suggestion
performance improvements for Trino queries using the supported object storage connectors. You'll hear
```

**Review by mosabua — CHANGES_REQUESTED** [review]

File: _episodes/55.md (hunk 1 above)
Comment (mosabua) [reply _episodes/55.md:47]: Also lets add Trino Gateway 5 - https://github.com/trinodb/trino-gateway/blob/main/docs/release-notes.md#trino-gateway-5-24-jan-2024

Comment (mosabua) [reply _episodes/55.md:47]: and charts 016, 0.17 and 0.18 - separate releases but just a few minor improvments

File: _episodes/55.md (hunk 3 above)
Comment (sakalsiz) [reply _episodes/55.md:53]: Yes, Peaka itself is a data stack. The text works for us, very nice.

File: _episodes/55.md [hunk 9]
```diff
@@ -0,0 +1,115 @@
...
+* Improved performance for Delta Lake queries without table statistics
+* Improved performance for Iceberg queries with filters on `ROW` columns
+
+[Trino 438](https://trino.io/docs/current/release/release-438.html)
+
+* Support for access control with [Open Policy Agent](https://trino.io/blog/2024/02/06/opa-arrived)
+* Support for `ALTER COLUMN ... DROP NOT NULL` in Iceberg and PostgreSQL
+* Support for configuring page sizes in Delta Lake, Hive, and Iceberg
+* Better type support for the `reduce_agg()` function
+* [Trino Gateway version 5](https://github.com/trinodb/trino-gateway/blob/main/docs/release-notes.md#trino-gateway-5-24-jan-2024)
```
Comment (mosabua) [inline _episodes/55.md:74]: This needs to move to a separate sentence or so after the 438 section. Its not related to 438

**Review by mosabua — APPROVED** [review]

**Review by mosabua — APPROVED** [review]
Looks good now.

**Pull request merged**